payments-core/src/main/java/com/stripe/android/IntentConfirmationInterceptor.kt

```diff
@@ -47,6 +47,11 @@ interface IntentConfirmationInterceptor {
         shippingValues: ConfirmPaymentIntentParams.Shipping?,
         setupForFutureUsage: ConfirmPaymentIntentParams.SetupFutureUsage?,
     ): NextStep
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    companion object {
+        var createIntentCallback: AbsCreateIntentCallback? = null
```

Comment:
We need to null this out in all cases, and we should add tests that confirm it's nulled out.

Reply:
Done

Reply:
Chatted out of band. This is likely incomplete. But we won't have a great way to test it until we start using it. Will follow up with more changes in this vein.